test/basic_parser.cpp

```diff
@@ -116,6 +116,19 @@ validate( string_view s )
     return true;
 }
 
+parse_options
+make_options(
+    bool comments,
+    bool commas,
+    bool utf8)
+{
+    parse_options opt;
+    opt.allow_comments = comments;
+    opt.allow_trailing_commas = commas;
+    opt.allow_invalid_utf8 = utf8;
+    return opt;
+}
```

Comment:
Why do you need this?  `parse_options{ false, true, false }` works with `parse_options` as you've currently defined it.

Reply:
@glenfe Not for C++11; `parse_options` has default member initializers and thus is not an aggregate. C++14 and above doesn't have this problem.